portfolio/src/main/java/com/google/sps/servlets/CommentHandlerServlet.java

```diff
@@ -6,23 +6,24 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.sps.apihelper.DatastoreHelper;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Safelist;
+
 @WebServlet("/suggestions")
 public class CommentHandlerServlet extends HttpServlet {
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         
-        //gets comment text
-        String comment = request.getParameter("text-comment");
+        //gets infromation
+        String comment = Jsoup.clean(request.getParameter("text-comment"), Safelist.none());
         String name = request.getParameter("commenter-name");
+        long timestamp = System.currentTimeMillis();
 
-        //log comment for later
-        System.out.println("Commenter: " + name);
-        System.out.println("Comment: " + comment);
-
-        //build response
-        response.setContentType("text/html;");
-        response.getWriter().println("Your comment was logged. Thank you, " + name + "!");
+        DatastoreHelper.write(name, comment, timestamp);
```

Comment:
Did you forget to include the new class in the PR? I don't see it.

Reply:
Yep it was -a issue. It should be added now, I used git add.